platforms/software/test-suites-base/src/main/java/org/gradle/testing/base/plugins/TestSuiteBasePlugin.java

```diff
@@ -16,9 +16,18 @@
 
 package org.gradle.testing.base.plugins;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Can we keep using org.apache.commons.lang.StringUtils? I'm not sure why we have both versions...

Reply:
Shouldn't we prefer lang3 over lang?

Reply:
Bah, it looks like we have a mix of both. I thought we were only using lang, not lang3

Reply:
I guess lang3 is good enough.  Ignore me